lms/static/scripts/frontend_apps/components/ContentSelector.tsx

```diff
@@ -47,6 +47,7 @@ export default function ContentSelector({
       d2l: { enabled: d2lFilesEnabled, listFiles: d2lListFilesApi },
       canvas: { enabled: canvasFilesEnabled, listFiles: listFilesApi },
       google: {
+        enabled: googleDriveEnabled,
```

Comment:
We could be smarter about these config types in future by omitting the settings object for a content type if it is not enabled or some required data is not present. That would reduce the number of nullish checks in the code and prevent mistakes where code tries to use a setting without checking the corresponding `enabled` flag.